src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -0,0 +1,345 @@
+        result.emplace(value.first, value.second.as<std::string>());
+    }
+    return result;
+}
+}  // namespace
+
+ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& model,
+                                             const std::shared_ptr<const ov::IPlugin>& plugin,
+                                             const ov::AnyMap& properties)
+    : ov::npuw::ICompiledModel(model, plugin),
+      m_options_desc(std::make_shared<::intel_npu::OptionsDesc>()),
+      m_cfg(m_options_desc) {
+    LOG_INFO("Creating LLMCompiledModel");
+    LOG_BLOCK();
+
+    ::intel_npu::registerNpuwLlmOptions(*m_options_desc);
+
+    std::map<std::string, ov::Any> npuw_llm_props;
+    std::map<std::string, ov::Any> other_props;
+    split_llm_properties(properties, npuw_llm_props, other_props);
+    m_cfg.update(any_copy(npuw_llm_props));
+
+    LOG_INFO("1. Creating kvcache model as clone of passed one.");
+    auto kvcache_model = model->clone();
+    LOG_INFO("2. Transform kvcache model from stateful to stateless.");
+    ov::pass::StatefulToStateless().run_on_model(kvcache_model);
+
+    LOG_INFO("3. Creating prefill model as clone of transformed kvcache one.");
+    auto prefill_model = kvcache_model->clone();
+    prefill_model->set_friendly_name(kvcache_model->get_friendly_name() + "_prefill");
+    LOG_INFO("4. Converting KV-cache in prefill model to FP16.");
+    prefill_model = cvt_kvcache_to_fp16(prefill_model);
+
+    LOG_INFO("5. Optimize kvcache kvcache model to output key/values for new token.");
+    kvcache_model = redirect_new_kv_to_output(kvcache_model);
+    LOG_INFO("6. Converting KV-cache in kvcache model to FP16.");
+    kvcache_model = cvt_kvcache_to_fp16(kvcache_model);
+
+    const uint32_t kMaxPromptLen = align_to(m_cfg.get<::intel_npu::NPUW_LLM_MAX_PROMPT_LEN>(), 64u);
+    const uint32_t kMinResponseLen = align_to(m_cfg.get<::intel_npu::NPUW_LLM_MIN_RESPONSE_LEN>(), 64u);
+    const ::intel_npu::npuw::llm::ModelDesc model_desc = m_cfg.get<::intel_npu::NPUW_LLM_MODEL_DESC>();
+    KVAxesPosition axes = get_kv_axes(model_desc.type);
+    m_kvcache_desc = KVCacheDesc{kMaxPromptLen, kMaxPromptLen + kMinResponseLen, 0u, axes.seq_len};
+    LOG_INFO("7. Make prefill model with static shapes");
+    reshape_to_static(prefill_model, m_kvcache_desc.max_prompt_size, m_kvcache_desc.max_prompt_size, axes);
+    LOG_INFO("8. Make kvcache model with static shapes");
+    reshape_to_static(kvcache_model, 1u, m_kvcache_desc.total_size, axes);
+
+    auto npudesc = extract_npu_descriptor(plugin);
+
+    ov::AnyMap properties_copy = other_props;
+    auto prefill_config = get_default_prefill_config(model, npudesc);
+    // NB: GENERATE_HINT is only applicable for default generate config!
+    const ::intel_npu::npuw::llm::GenerateHint generate_hint = m_cfg.get<::intel_npu::NPUW_LLM_GENERATE_HINT>();
+    LOG_INFO("9. Passed GENERATE_HINT: " << ::intel_npu::NPUW_LLM_GENERATE_HINT::toString(generate_hint));
```

Comment:
Did you have these logs here before? Did GenAI produce these? Let's not add much verbosity into this, INFO is already a quite polluted log level which needs to be cleaned-up